Documentation/workflow/UsingYourBuild.md

```diff
@@ -46,148 +54,158 @@ this by simply listing the name of the Microsoft.NETCore.Runtime.CoreCLR you bui
-Thus if you modify the source and create a new NuGet package you must it a new version number and use that in your 
-application's project.json.   Otherwise the dotnet.exe tool will assume that the existing version is fine and you 
+Thus if you modify the source and create a new NuGet package you must it a new version number and use that in your
+application's project.   Otherwise the dotnet.exe tool will assume that the existing version is fine and you
 won't get the updated bits.   This is what the Minor Build number is all about.  By default it is 0, but you can
-give it a value by setting the BuildNumberMinor environment variable. 
+give it a value by setting the BuildNumberMinor environment variable.
 ```bat
     set BuildNumberMinor=3
 ```
-before packaging.   You should see this number show up in the version number (e.g. 1.2.0-beta-24521-03).
+before packaging.   You should see this number show up in the version number (e.g. 2.0.0-beta-25023-03).
 
-As an alternative you can delete the existing copy of the package from the Nuget cache.   For example on 
-windows (on Linux substitute ~/ for %HOMEPATH%) you could delete 
+As an alternative you can delete the existing copy of the package from the Nuget cache.   For example on
+windows (on Linux substitute ~/ for %HOMEPATH%) you could delete
 ```bat
-     %HOMEPATH%\.nuget\packages\Microsoft.NETCore.Runtime.CoreCLR\1.2.0-beta-24521-02
+     %HOMEPATH%\.nuget\packages\Microsoft.NETCore.Runtime.CoreCLR\2.0.0-beta-25023-02
 ```
-which should make things work (but is fragile, confirm wile file timestamps that you are getting the version you expect)
+which should make things work (but is fragile, confirm file timestamps that you are getting the version you expect)
 
 
-## Step 8.1 (Optional) Quick updates in place.  
+## Step 9.1 (Optional) Quick updates in place.
 
-The 'dotnet publish' step in step 6 above creates a directory that has all the files necessary to run your app
+The 'dotnet publish' step  above creates a directory that has all the files necessary to run your app
 including the CoreCLR and the parts of CoreFX that were needed.    You can use this fact to skip some steps if
 you wish to update the DLLs.   For example typically when you update CoreCLR you end up updating one of two DLLs
 
 * coreclr.dll - Most modifications (with the exception of the JIT compiler and tools) that are C++ code update
-  this DLL. 
-* System.Private.CoreLib.dll - If you modified C# it will end up here.  
+  this DLL.
+* System.Private.CoreLib.dll - If you modified C# it will end up here.
 * System.Private.CoreLib.ni.dll - the native image (code) for System.Private.Corelib.   If you modify C# code
-you will want to update both of these together in the target installation.   
+you will want to update both of these together in the target installation.
 
 Thus after making a change and building, you can simply copy the updated binary from the `bin\Product\<OS>.<arch>.<flavor>`
```

Comment:
I think we should be better to recommend to just copy the whole directory - it is what works best. It is not easy to pick the right subset correctly, and the crashes that results from inconsistent set are hard to debug.